pkg/util/vdbutil/parser.go

```diff
@@ -0,0 +1,59 @@
+/*
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to You under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package vdbutil
+
+import (
+	"regexp"
+	"strings"
+)
+
+// DatasourceInfo --
+type DatasourceInfo struct {
+	Name string `yaml:"name,omitempty"`
+	Type string `yaml:"type,omitempty"`
+}
+
+// Tokenize --
+func Tokenize(ddl string) []string {
+	a := regexp.MustCompile(";")
```

Comment:
@shawkins this is one place I do need your help, I could not come up a delimiter for separating the statements where I need to escape semi-colon (;). Can you come with a regex here?

Reply:
The parsing of statements and even the create server can be arbitrarily complex if we fully account for things like comments as well (both inline and multi-line).

I tend to think in terms of a Matcher, something like this:

``
        String pattern = "(?is)\\\\w('[^']*'|\\"[^\\"]*\\"|[^'\\";])*;";
        String foo = "not a create; CREATE server ... 2;create server ... 1;' create ';";
        Pattern p = Pattern.compile(pattern);
        Matcher m = p.matcher(foo);

        while (m.find()) {
            System.out.println(m.group());
        }
``

Comments are not accounted for, but this will find everything that looks like a statement by trying to keep quoted stuff paired.